compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -11,6 +11,8 @@ class JSExportTest extends DirectTest with TestHelpers {
 
   @Test
   def noDoubleUnderscoreExport = {
+    // scalastyle:off
```

Comment:
Oh, found it: http://www.scalastyle.org/configuration.html
There's a discrete paragraph about that ^^

Well, I'll update all the off's and on's to disable the specific rule which is annoying, then.